tiny_dnn/activations/activation_layer.h

```diff
@@ -101,6 +101,11 @@ class activation_layer : public layer {
                                    vec_t &dx,
                                    const vec_t &dy) = 0;
 
+  /**
+   * Target value range for learning.
+   */
+  virtual std::pair<float_t, float_t> scale() const = 0;
```

Comment:
@edgarriba An advantage of making base method pure virtual is new activation function layer class writer would pay attention to important scale value. (Comment says it is target value range for learning, but it's all Greek to me...)
Contrary, an advantage of making base method non-pure virtual is implementors can have the luxury of object-oriented programming language.